packages/api/src/AwilixHelpers.ts

```diff
@@ -61,3 +64,39 @@ function makeProxyAwilixFriendly(
     return prefix + " " + result;
   };
 }
+
+/**
+ * Apply the ContainerAware mixin to this class. You must provide this.container
+ * @provides buildMethod
+ * @example
+ * ```
+ * @MakeContainerAware()
+ * export class SomeClass {
+ *
```

Comment:
This example should be finished

Reply:
This whole structure just reeks. Not going to document in favor of getting rid of it. Need to find a different path forward for the auth services.

Reply:
opened #100